docs/index.md

```diff
@@ -0,0 +1,52 @@
+# cql2-rs
+
+**cql-rs** is command-line interface (CLI), Python package, and Rust crate for parsing, validating, and converting [Common Query Language (CQL2)](https://www.ogc.org/standard/cql2/).
+
+## CLI
+
+To install the CLI, you'll need [Rust](https://rustup.rs/).
+Then:
+
+```shell
+$ cargo install cql2-cli
+$ cql2 -o json "landsat:scene_id = 'LC82030282019133LGN00'"
+{"op":"=","args":[{"property":"landsat:scene_id"},"LC82030282019133LGN00"]}
+```
+
+## Python
+
+Install with **pip**:
+
+```shell
+pip install cql2
```

Comment:
```shell
uv pip install cql2
```

😉 

I've seen you work w/ the `python -m pip` pattern before, can you point me to the background on it? No objections from my side, just curiosity 🙇🏼

Reply:
the issue is that `pip` command line could point to some obscure python installation (usually when working with virtualenv) 

doing `python -m` guarantees that you're running the `desired` python environment. I don't remember who told me to do this (maybe Sean G 🤷)

Reply:
`python -m pip` is indeed safer to use than `pip` directly, in the case that the `pip` on the `PATH` is a different Python than `python`. But since this is rare, I usually suggest `pip install` in my own docs for brevity